core/pta/stats.c

```diff
@@ -209,6 +221,28 @@ static TEE_Result get_system_time(uint32_t type,
 	return TEE_SUCCESS;
 }
 
+static TEE_Result get_drivers_info(uint32_t type, TEE_Param p[TEE_NUM_PARAMS])
```

Comment:
`print_driver_info()`. This does not get anything.
...which leads to the question: should a buffer be returned to the client instead? That would be more consistent with other statistics and also would be usable when `CFG_TEE_CORE_LOG_LEVEL < TRACE_INFO`.

Reply:
Right, I'll change the function label.
Note that command `STATS_CMD_MEMLEAK_STATS` already behaves this way.

Reply:
> Note that command `STATS_CMD_MEMLEAK_STATS` already behaves this way.

Fair enough. Printing to the secure console is probably good enough for the main use cases (debugging).